addons/website_slides/models/slide_slide.py

```diff
@@ -872,6 +1054,18 @@ def _default_website_meta(self):
     # Data / Misc
     # ---------------------------------------------------------
 
+    def _get_completion_time_pdf(self, data):
```

Comment:
b64_data maybe ? Or tell it shoudl be base64 encoded ?

Reply:
Actually not sure here, I think we pass the method bytes, not base64 encoded stuff.